cvmfs/swissknife_check.cc

```diff
@@ -369,12 +385,13 @@ bool CommandCheck::Find(const catalog::Catalog *catalog,
     }
 
     // Check if the chunk is there
-    if (check_chunks_ &&
-        !entries[i].checksum().IsNull() && !entries[i].IsExternalFile())
+    if (check_chunks_ && entry_needs_check)
     {
       string chunk_path = "data/" + entries[i].checksum().MakePath();
       if (entries[i].IsDirectory())
         chunk_path += shash::kSuffixMicroCatalog;
+    LogCvmfs(kLogCvmfs, kLogVerboseMsg, "[chunkpath] %s",
```

Comment:
- I guess here is a whitespace error
- The word "chunk" is now actually used for the chunks of a multi-part file. To avoid confusion, I' suggest `[object path]`

Reply:
Yes,  this was only  a debug printout for myself, I'm not sure it's needed for the logs, I'll remove it alltogether